scss/texts.scss

```diff
@@ -0,0 +1,11 @@
+$gray : #828282;
```

Comment:
Move this to a `colors.scss` file and import it here

Reply:
Done